README.md

```diff
@@ -1,7 +1,37 @@
 ## Download Files From S3
 
-This will be a lambda which will take the messages from the S3 uploads and download them to the EFS volume, where they can be reused by other processes.
+This is a lambda which takes the messages from the S3 uploads and downloads them to the EFS volume, where they can be reused by other processes.
```

Comment:
Is it worth adding a line in here to say that the lambda also sends messages to each of the file check queues? I missed it out when I first did this.

Reply:
Done, thanks. I've also linked to the file check ADR and architecture diagram for more context.